app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,48 @@
+.container {
+  margin: 48px auto;
+  width: 960px;
+
+  section {
+    clear: both;
+  }
+
+  p {
+    font-size: 22px;
+  }
+
+  header {
+    margin-bottom: 46px;
+
+    &.clearfix:after {
+      clear: both;
+      content: "";
+      display: block;
+      height: 0;
+      line-height: 0;
+      visibility: hidden;
+    }
+  }
+
+  header h1 {
```

Comment:
Merge rule `header h1` with rule on line 13<br>Selector should have depth of applicability no greater than 2, but was 3